File: src/main/java/com/sunnyxiaobai5/web/home/HomeController.java
/*******************************************************************************
 * Project: spring-data-demo
 * FileName:HomeController.java 
 * Author:  Xiangyong Zeng
 * Date:    2015/10/12 16:11
 * Copyright: 2015 www.yineng.com.cn Inc. All rights reserved.
 * Description:
 ******************************************************************************/

package com.sunnyxiaobai5.web.home;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;

/**
 * Project: spring-data-demo
 * Package: com.sunnyxiaobai5.web.home
 * Title:   描述
 * Author:  Xiangyong Zeng
 * Date:    2015/10/12 16:11
 * Version: 1.0
 */
@Controller
//@RequestMapping("/")
public class HomeController {

    @RequestMapping("/homePage")
    String homePage() {
        return "/index.jsp";
    }

    @RequestMapping("/homeInfo")
    @ResponseBody
    String homeInfo(){
        return "home info String";
    }


    @RequestMapping("/home")
    @ResponseBody
    String home() {
        return "hello home";
    }
}

File: src/main/webapp/scripts/app/home/home.controller.js
/*******************************************************************************
 * sunnyxiaobai5@gmail.com
 *
 * <li>项目名称: spring-data-demo</li>
 * <li>文件名称: home.controller.js</li>
 * <li>内容摘要: </li>
 * <li>内容描述: </li>
 * <li>其他说明: </li>
 * <li>@author Xiangyong Zeng</li>
 ******************************************************************************/
(function () {
    'use strict';

    angular.module('elapseApp').controller('HomeController', ['$scope', '$state', 'Menu', function ($scope, $state, Menu) {
        $scope.systems = [];
        Menu.findSystems(function (result) {
            angular.forEach(result, function (system) {
                $scope.systems.push(system);
            });
        });
    }]);
})();
